src/deploy/firestore/release.ts

```diff
@@ -1,19 +1,23 @@
 import * as _ from "lodash";
 
 import { RulesDeploy, RulesetServiceType } from "../../rulesDeploy";
+import { Options } from "../../options";
+import { FirebaseError } from "../../error";
 
 /**
  * Releases Firestore rules.
  * @param context The deploy context.
  * @param options The CLI options object.
  */
-export default async function (context: any, options: any): Promise<void> {
+export default async function (context: any, options: Options): Promise<void> {
   const rulesDeploy: RulesDeploy = _.get(context, "firestore.rulesDeploy");
   if (!context.firestoreRules || !rulesDeploy) {
     return;
   }
-  await rulesDeploy.release(
-    options.config.get("firestore.rules"),
-    RulesetServiceType.CLOUD_FIRESTORE
-  );
+
+  const rulesFile = options.config.src.firestore?.rules;
+  if (!rulesFile) {
+    throw new FirebaseError(`Invalid firestore config: ${options.config.src.firestore}`);
```

Comment:
I _think_ this may print as "Invalid firestore config [Object object]" - can you double check and stringify if needed?